tools/python/google_maps_bookmarks.py

```diff
@@ -2,97 +2,133 @@
 
 import csv
 import json
-import os.path
+import argparse
+import mimetypes
 import traceback
 import urllib.error
 import urllib.parse
 import urllib.request
 import xml.etree.ElementTree as ET
-from os import R_OK
+from os import path, access, R_OK, linesep
+from io import StringIO
 
 
-class GoogleMapsConverter:
+from dateutil import parser as dateutil_parser
 
-    def __init__(self):
-        print("Follow these steps to export your saved places from Google Maps and convert them to a KML-File")
-        print()
-        print("1. Create an API key for Google Places API following this guide")
+class GoogleMapsConverter:
+    def __init__(self, input_file, output_format):
+        print("Follow these steps to export your saved places from Google Maps and convert them to a GPX or KML File")
         print()
-        print("https://developers.google.com/maps/documentation/places/web-service/get-api-key")
+        print("OPTION 1: Using GeoJSON")
+        print("===================================")
+        print("1. Go to \"Your data in Maps\" in your Google Account settings or by accessing https://myaccount.google.com/yourdata/maps")
```

Comment:
I think these instructions could confuse the users of this script. They just want to export their bookmarks from Google Maps, so their probably not thinking in a GeoJSON/CSV workflow. If you follow the initial instructions, you also get the GeoJSON file from Google Takeout. You also need an API-Key for GeoJSON (see https://github.com/organicmaps/organicmaps/pull/8721#pullrequestreview-2247250563). So I would keep the initial instructions and add a step 4a./4b. that explains how the script is called with a csv or geojson file.